http/swagger.yml

```diff
@@ -7510,6 +7517,32 @@ components:
           type: string
         check:
           $ref: '#/components/schemas/Check'
+        queries:
+          type: array
+          items:
+            $ref: "#/components/schemas/DashboardQuery"
+        colors:
+          description: Colors define color encoding of data into a visualization
+          type: array
+          items:
+            $ref: "#/components/schemas/DashboardColor"
+        note:
+          type: string
+        showNoteWhenEmpty:
+          description: if true, will display note when empty
+          type: boolean
+        axes:
```

Comment:
I think this is out of scope for this PR, I have created an issue to address it https://github.com/influxdata/influxdb/issues/14560

Reply:
It will be much easier to change this for the `CheckView` than for the `XYView` (since the `XYView` requires a migration), so I think they are somewhat separate. I also think it's worth addressing the `CheckView` now, since this is when we are first introducing it's schema. Otherwise we'll build off of it and make the change harder for ourselves in the future. But up to you!